docs/output.md

```diff
@@ -171,4 +171,19 @@ osv-scanner --format json -L path/to/lockfile > /path/to/file.json
 }
 ```
 
-</details>
\ No newline at end of file
+</details>
+
+## Return Codes
```

Comment:
nit: I feel like this might look nicer in a table

Reply:
Good idea, done!